### PR TITLE
add Date to print page.

### PR DESCRIPTION
**Pull request by rembiesa**

### CODE REVIEW

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #16155 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/16155)

File: layouts/basic/modules/Vtiger/Modals/RecordCollector.tpl [hunk 1]
```diff
@@ -34,6 +34,9 @@
 			</div>
 		</form>
 		<div class="mt-5 pt-2 js-record-collector__summary" data-js="html"></div>
+		<div class="js-printed-by" style="display: none">
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Vtiger/Modals/RecordCollector.tpl]: class d-none

File: languages/en-US/Other/RecordCollector.json [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {
   "php": {
-    "LBL_POLAND_VAT_PAYER": "Verification of the status of the VAT payer",
-    "LBL_POLAND_VAT_PAYER_DESC": "Verifies whether the entity is registered as a VAT taxpayer on prior to sending the inquiry based on the tax identification number.",
-    "LBL_POLAND_GUS": "GUS",
-    "LBL_POLAND_GUS_DESC": "Central Statistical Office of Poland",
-    "LBL_VIES": "VIES",
-    "LBL_VIES_DESC": "The VAT Information Exchange System (VIES) is an electronic means of transmitting information relating to VAT registration (i.e., validity of VAT numbers) of companies registered in the European Union."
+    "LBL_POLAND_VAT_PAYER": "Verify VAT taxpayer’s [PL] status ",
```
Comment (mariuszkrzaczkowski) [inline languages/en-US/Other/RecordCollector.json]: ?

File: modules/Vtiger/views/RecordCollector.php [hunk 3]
```diff
@@ -54,6 +54,7 @@ public function process(App\Request $request)
 		$viewer->assign('RECORD_ID', $request->getInteger('record'));
 		$viewer->assign('RECORD_COLLECTOR', $this->recordCollector);
 		$viewer->assign('COLLECTOR_NAME', $request->getByType('collectorType', 'ClassName'));
+		$viewer->assign('USER_MODEL', \App\User::getUserModel(\App\User::getCurrentUserId()));
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/views/RecordCollector.php]: remove

File: public_html/layouts/resources/app.js [hunk 4]
```diff
@@ -3458,11 +3458,22 @@ var app = (window.app = {
 		container.on('click', '.js-print-container', function (event) {
 			event.preventDefault();
 			const element = $(this);
+			let printedBy = $('.js-printed-by');
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/resources/app.js]: element.closest

**Review by mariuszkrzaczkowski — CHANGES_REQUESTED** [review]

File: layouts/basic/modules/Vtiger/Modals/RecordCollector.tpl [hunk 5]
```diff
@@ -34,6 +34,9 @@
 			</div>
 		</form>
 		<div class="mt-5 pt-2 js-record-collector__summary" data-js="html"></div>
+		<div class="js-printed-by d-none d-print-block">
+			{\App\Language::translate('LBL_PRINTED_BY', 'Other.RecordCollector')} {$USER_MODEL->getName()} {\App\Language::translate('LBL_IN_DAY', 'Other.RecordCollector')} {date($DATE_FORMAT)}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Vtiger/Modals/RecordCollector.tpl]: date($DATE_FORMAT) >> App\Fields\DateTime::formatToDisplay('now')

**Pull request merged**